app/controllers/users/registrations_controller.rb

```diff
@@ -10,6 +10,9 @@ def create
         fridge = Fridge.new(name: 'My Fridgy')
         fridge.user = resource
         fridge.save!
+        list = List.new(name: 'My Fridgy')
```

Comment:
My last branch i've updated the name to My Fridgi

Reply:
Sorry, forgot to split my branch did not finish the list part :)